PEP585: More UP006 fixes

aorenste opened the pull request:
This should be the final PR before we can enable RUFF UP006.

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #146392



cc @H-Huang @awgu @kwen2501 @wanchaol @fegin @fduwjj @wz337 @wconstab @d4l3k @c-p-i-o @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @amjames @chauhang @aakhundov @LucasLLC @MeetVadakkanchery @mhorowitz @pradeepfn @ekr0

aorenste committed:
PEP585: More fixes

[ghstack-poisoned]

aorenste committed:
Update on "PEP585: More fixes"




[ghstack-poisoned]

aorenste committed:
Update on "PEP585: More fixes"




cc H-Huang awgu kwen2501 wanchaol fegin fduwjj wz337 wconstab d4l3k c-p-i-o voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 amjames chauhang aakhundov LucasLLC MeetVadakkanchery mhorowitz pradeepfn ekr0

[ghstack-poisoned]

aorenste committed:
Update on "PEP585: More fixes"




cc H-Huang awgu kwen2501 wanchaol fegin fduwjj wz337 wconstab d4l3k c-p-i-o voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 amjames chauhang aakhundov LucasLLC MeetVadakkanchery mhorowitz pradeepfn ekr0

[ghstack-poisoned]

justinchuby reviewed (approved):
onnx changes lgtm, thanks!

aorenste committed:
Update on "PEP585: More UP006 fixes"

This should be the final PR before we can enable RUFF UP006.




cc H-Huang awgu kwen2501 wanchaol fegin fduwjj wz337 wconstab d4l3k c-p-i-o voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 amjames chauhang aakhundov LucasLLC MeetVadakkanchery mhorowitz pradeepfn ekr0

[ghstack-poisoned]

albanD reviewed (approved):
Thanks!

aorenste committed:
Update on "PEP585: More UP006 fixes"

This should be the final PR before we can enable RUFF UP006.




cc H-Huang awgu kwen2501 wanchaol fegin fduwjj wz337 wconstab d4l3k c-p-i-o voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 amjames chauhang aakhundov LucasLLC MeetVadakkanchery mhorowitz pradeepfn ekr0

[ghstack-poisoned]

aorenste commented:
@pytorchbot merge
